### PR TITLE
Remove pdf mentions & build rules

### PR DESCRIPTION
**Pull request by hflynn**
See https://trello.com/c/SnUesylG/505-ditching-docs-pdfs for context

Doc changes staged at http://www.openmicroscopy.org/site/support/contributing-staging/ci-docs.html

@rleigh-codelibre can i delete common/preamble.tex too? Is there anything else I've missed?

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -147,8 +114,6 @@ The output should look something like::
       qthelp     to make HTML files and a qthelp project
       devhelp    to make HTML files and a Devhelp project
       epub       to make an epub
-      latex      to make LaTeX files, you can set PAPER=a4 or PAPER=letter
-      latexpdf   to make LaTeX files and run them through pdflatex
```
Comment (rleigh-codelibre) [inline README.rst:151]: These can be retained, particularly if we're retaining all other types like epub, devhelp and the rest.  This is a list of all possible types, and latex will remain a possible type (which we don't use, like all the rest here).

File: README.rst [hunk 2]
```diff
@@ -175,10 +140,6 @@ To build the sets of documentation locally in the form of HTML pages, use one of
     make html
     ant html
 
-To build the sets of documentation locally in the form of a PDF file, use one of::
-
-    make latexpdf
-    ant latexpdf
```
Comment (rleigh-codelibre) [inline README.rst]: Remove extra newline?

File: README.rst [hunk 3]
```diff
@@ -354,8 +315,7 @@ Tables
 ======
 
 Please do not use tables for collections of links and figures, and leave them
-solely for use as actual tables. While it can be used in HTML to shoehorn
-content into boxes, it does not work too well for other output, e.g. latex.
+solely for use as actual tables.
```
Comment (rleigh-codelibre) [inline README.rst]: This is maybe still recommended advice even when not using latex?

**Comment by rleigh-codelibre** [comment]
All looks good, some comments noted.

**Comment by rleigh-codelibre** [comment]
You can also remove latex configuration from conf.py, like so:

```
diff --git a/common/conf.py b/common/conf.py
index 8ccac6d..e3a7b08 100644
--- a/common/conf.py
+++ b/common/conf.py
@@ -293,31 +293,18 @@ htmlhelp_basename = 'projstandardsdoc'
 
 # -- Options for LaTeX output --------------------------------------------------
 
-latex_elements = {
-    'classoptions': ',oneside',
-    'pointsize': '10pt',
-    'inputenc': '%% Unused',
-    'utf8extra': '%% Unused',
-    'fontenc' : '%% Unused',
-    'fontpkg': '%% Unused',
-    'babel': '%% Unused',
-    'printindex': '''\\phantomsection
-\\addcontentsline{toc}{part}{\indexname}
-\\printindex''',
-    'preamble': '''
-\input{../../../common/preamble.tex}
-''',
-}
+#latex_elements = {
+#}
 
 # For "manual" documents, if this is true, then toplevel headings are parts,
 # not chapters.
-latex_use_parts = True
+#latex_use_parts = True
 
 # If true, show page references after internal links.
 #latex_show_pagerefs = False
 
 # If true, show URL addresses after external links.
-latex_show_urls = 'footnote'
+#latex_show_urls = 'footnote'
 
 # Documents to append as an appendix to all manuals.
 #latex_appendices = []
```

(this is already done in the ome-model conf.py)

**Comment by hflynn** [comment]
@rleigh-codelibre made those changes, thanks. you didn't comment on preamble.tex as I mentioned above - that still serves some purpose, does it?

**Comment by rleigh-codelibre** [comment]
Oops, forgot to reply.  No, `preamble.tex` can go as well; it's purely settings for latex.

File: omero/conf.py [hunk 4]
```diff
@@ -107,20 +107,6 @@
 # Add any paths that contain custom themes here, relative to this directory.
 html_theme_path.extend(['themes'])
 
-# -- Options for LaTeX output --------------------------------------------------
```
Comment (rleigh-codelibre) [inline omero/conf.py:110]: I would leave the template in place, but empty, as I did here: https://github.com/ome/ome-model/blob/master/docs/sphinx/conf.py#L304

Then the template structure is in place as for all the other bits of conf.py we don't use.

Comment (hflynn) [reply omero/conf.py:110]: Ah, when you said remove I assumed delete rather than comment out - I'll fix that now

**Comment by hflynn** [comment]
Hopefully sorted now @rleigh-codelibre

File: common/conf.py [hunk 5]
```diff
@@ -290,42 +290,40 @@ def get_previous_version(majornumber, minornumber=0):
...
+# \\printindex''',
+#    'preamble': '''
+# \input{../../../common/preamble.tex}
+# ''',
+# }
 
 # For "manual" documents, if this is true, then toplevel headings are parts,
 # not chapters.
-latex_use_parts = True
+# latex_use_parts = True
```
Comment (rleigh-codelibre) [inline common/conf.py]: You don't need a space after (for consistency with other settings here, and in the other repos), same for the setting below

File: omero/conf.py [hunk 6]
```diff
@@ -111,15 +111,14 @@
 
 # Grouping the document tree into LaTeX files. List of tuples
 # (source start file, target name, title, author, documentclass [howto/manual]).
-target = project + '-' + release + '.tex'
-latex_documents = [
-  (master_doc, target, title, author, 'manual'),
-]
+# target = project + '-' + release + '.tex'
```
Comment (rleigh-codelibre) [inline omero/conf.py]: All the `# ` here can be `#` i.e. no space after the hash.

**Comment by rleigh-codelibre** [comment]
If you would like to cherry-pick the last commit from my pdf-removal branch, that takes care of a few other remaining references I saw.  42a2bec635cff3938c5fa16091568afeaa8e4bca

As a followup task, `.. tabularcolumns::` can be removed from every file, which is the latex table formatting.  Edit: they are only in a few files, in

```
% git grep -l tabularcol
omero/developers/Modules/Search.txt
omero/sysadmins/server-permissions.txt
omero/sysadmins/system-requirements.txt
omero/sysadmins/unix/server-installation.txt
omero/sysadmins/version-requirements.txt
```

so could be removed now if you like.

**Comment by hflynn** [comment]
Rebuilding OMERO merge docs since I've touched a bunch of files there too, should be fine though, travis green and it's fine locally

**Comment by hflynn** [comment]
https://ci.openmicroscopy.org/view/Docs/job/OMERO-DEV-merge-docs/559/ is green

**Comment by rleigh-codelibre** [comment]
All looks fine with me.  Please merge if all the jobs are green.

**Comment by hflynn** [comment]
Just rebuilding the contributing docs too to be absolutely certain all the affected builds are fine

**Comment by hflynn** [comment]
Everything is green this morning

**Review by sbesson — APPROVED** [review]

**Pull request merged**